ocean_lib/models/arguments.py

```diff
@@ -0,0 +1,166 @@
+#
```

Comment:
yes, but these are shared: we would have circular import errors btw. DataNFTFactory, DataNFT and Datatoken

Reply:
OK. 

Then we have three choices:
1. local JIT import, inside methods, to avoid. Though I don't think that really works here
2. status quo of this PR: put them in arguments.py
3. a _new_ module named DataNFTArguments, and another DatatokenArguments

(1) isn't really on the table. So it's (2) or (3). I prefer (3). Less "global", especially if we repeat the pattern for other modules.